libbeat/outputs/kafka/kafka_integration_test.go

```diff
@@ -57,11 +59,13 @@ func newTestKafkaClient(t *testing.T, topic string) *client {
 
 func newTestKafkaOutput(t *testing.T, topic string, useType bool) outputs.Outputer {
 
+	if useType {
```

Comment:
use_type has been removed from config. This is in order to keep the tests working without much change, as some tests use the event `type` to choose topic dynamically and other tests set static topic name. Alternatively one can change tests to pass format-string in `topic` and remove `useType`.